lib/presentation/crowdaction/crowdaction_comments/widgets/comment_delete_button.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+///import '../../../core/collaction_icons.dart';
+
+class CommentDeleteButton extends StatelessWidget {
+  final Function()? onTap;
+
+  const CommentDeleteButton({
+    super.key,
+    this.onTap,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return IconButton(
+      /// TODO: replace icons with variants from CollactionIcons
```

Comment:
We use IonIcons, you should be able to find it, it's in the project most likely just not used anywhere.

Reply:
I've changed it now, though I couldn't find a similar flagging icon to the one in the figma, so I've used Ionicons.flag. Is that alright for now?

Reply:
Yes that is what I meant 👍